README.md

```diff
@@ -42,4 +42,6 @@ yields:
    password=bar
    ```
 
-2. Publish a new release with `sbt publishSigned`
+2. Publish a new release with `sbt publish`
```

Comment:
I think for now we should remove this, since we don't want people overwriting the auto-published jars.